v2/concordium/types.proto

```diff
@@ -1578,8 +1599,8 @@ message TransactionFeeDistribution {
   AmountFraction gas_account = 2;
 }
 
-// Distribution of gas rewards.
-message GasRewards {
+// Distribution of gas rewards for chain parameters version 0 and 1.
+message GasRewardsCpv0 {
```

Comment:
This renaming does not seem needed other than to satisfy some artificial standard of purity.

It is not a lot of work, but it is needless work caused by this renaming.

Reply:
Changed it back.